pandas/tests/tseries/test_timezones.py

```diff
@@ -1280,6 +1280,25 @@ def test_ambiguous_compat(self):
             self.assertEqual(result_pytz.to_pydatetime().tzname(),
                              result_dateutil.to_pydatetime().tzname())
 
+    def test_tzreplace_issue_15683(self):
+        """Regression test for issue 15683."""
```

Comment:
Usually we just leave a comment on the first line of the function like `# GH 15683`